process/process.h

```diff
@@ -0,0 +1,8 @@
+#ifndef PROCESS_H
+#define PROCESS_H
+#include <string>
+#include <iostream>
+
+void process_command(const std::string &command);
```

Comment:
Move `process_command` and it's definition to cmd.h and cmd.cpp. There's no reason for this to be in a different header file if it forms part of the cmd program.